website/src/data/top-menu.yaml

```diff
@@ -1,28 +1,28 @@
 - menu-title: Composable Commerce
   items:
     - label: Getting Started
-      href: https://docs.commercetools.com/getting-started/
+      href: /../getting-started/
     - label: Merchant Center
-      href: https://docs.commercetools.com/merchant-center
+      href: /../merchant-center
     - label: Tutorials
-      href: https://docs.commercetools.com/tutorials
+      href: /../tutorials
     - label: HTTP API
-      href: https://docs.commercetools.com/api
+      href: /../api
     - label: GraphQL API
-      href: https://docs.commercetools.com/api/graphql
+      href: /../api/graphql
     - label: Import & Export
-      href: https://docs.commercetools.com/import-export
+      href: /../import-export
     - label: SDKs & Client Libraries
-      href: https://docs.commercetools.com/sdk
+      href: /../sdk
     - label: Custom Applications
-      href: https://docs.commercetools.com/custom-applications
```

Comment:
@timonrey  for purposes of using the previews in the app kit docs deployment the links could have been remaining absolute here.   The other microsites are not deployed together with this one so a preview site (e.g. private beta) would have broken top nav links.

Reply:
@nkuehn Yes, the problem was that the new feature for the top bar showing `Composable Commerce > Custom applications` was not working with absolute links, they needed to be relative paths. We can adjust the feature and change it into absolute links separately.

Reply:
Right.  Thank you for the clarification!  It#s an edge case, OK that way.